partials/navbar.html

```diff
@@ -20,6 +20,12 @@
         <i class="glyphicon glyphicon-chevron-right"></i>
       </a>
     </li>
+    <li>
+      <a href="#nav-eligibility" data-ng-click="scrollTo('nav-eligibility')">
```

Comment:
Here you use `nav-eligibility` as the scrollto target, but above in `partials/eligibility.html`, you have `nav-credits` as the `id` attribute value of your `<h2>` tag. They don't match, so clicking on this link in the sidebar doesn't work correctly. Either id would work, but they must match

Reply:
Fixed.